torch/_torch_docs.py

```diff
@@ -1130,6 +1130,31 @@ def merge_dicts(*dicts):
     tensor([ True, False, False])
 """.format(**common_args))
 
+add_docstr(torch.broadcast_to,
+           r"""
+broadcast_to(input, size) -> Tensor
+
+Broadcasts the input tensor to a new shape according to :ref:`broadcasting-semantics`.
+
+.. warning::
```

Comment:
This warning can be removed once expand is cited.

Reply:
Addressed.